src/autocapture.ts

```diff
@@ -86,6 +86,13 @@ export class Autocapture {
         }
     }
 
+    public startOrStopIfEnabled() {
```

Comment:
`startOrStopIfEnabled` is a bit of a mouthful though 🙈 

It's more really `startIfEnabledOrMaybeStop` which isn't any less of a mouthful

I can't think of a better name though so 🤷

Reply:
Looks like session recording is the only one that calls stop.

It alway seemed odd to me that it does. So maybe it shouldn't?

Then these could all be `startIfEnabled`

Reply:
What I was going for rightly or wrongly is to try and have some standard naming so that one is perhaps reminded to consider both the decide response _and_ the config change otherwise it will continue to get forgotten about...

Reply:
`onDecideOrConfig`?

But tbh I don't have a better name and we can change it in future so not the worst thing that ever happened 😊

Reply:
I came around to your thinking 👍